packages/gdrcopy.spec

```diff
@@ -27,7 +27,7 @@
 %endif
 
 %define gdrdrv_install_script                                           \
-/sbin/depmod -a %{kernel_version} &> /dev/null ||:                      \
+/sbin/depmod -a %{kernel_version}%{?dist}.%{_arch} &> /dev/null ||:                      \
```

Comment:
so it's broken in general? Maybe also make PR to upstream?

Reply:
As discussed, change of strategy. We'll follow upstream.